ehsm_kms_service/kubernetes/ehsm-kms.yaml

```diff
@@ -0,0 +1,203 @@
+# ehsm-kms Secret
+apiVersion: v1
+kind: Secret
+metadata:
+    name: ehsm-secret
+    namespace: ehsm-kms
+type: Opaque
+data:
+    couch_root_username: YWRtaW4=
+    couch_root_password: cGFzc3dvcmQ=
+
+---
+# ehsm-kms ConfigMap
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ehsm-configmap
+  namespace: ehsm-kms
+data:
+  database_url: "couchdb-0.couchdb"
+  database_port: "5984"
+  database_name: "ehsm_kms_db"
+
+---
+# ehsm-kms PersistentVolume for CouchDB
+apiVersion: v1
+kind: PersistentVolume 
+metadata:
+  name: ehsm-pv-nfs
+spec:
+  capacity:
+    storage: 10Gi 
+  accessModes:
+    - ReadWriteOnce 
+  persistentVolumeReclaimPolicy: Retain 
+  storageClassName: nfs
+  nfs:
+    path: /nfs_ehsm_db
+    server: 1.2.3.4
+
+---
+# ehsm-kms CouchDB
+apiVersion: v1
+kind: Service
+metadata:
+  name: couchdb
+  namespace: ehsm-kms
+  labels:
+    app: couchdb
+spec:
+  ports:
+    - name: couchdb
+      port: 5984
+      targetPort: 5984
+  selector:
+    app: couchdb
+  clusterIP: None 
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: couchdb
+  namespace: ehsm-kms
+spec:
+  selector:
+    matchLabels:
+      app: couchdb
+  serviceName: "couchdb"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: couchdb
+    spec:
+      containers:
+      - name: couchdb
+        image: couchdb:latest
+        imagePullPolicy: IfNotPresent
+        readinessProbe:
+          httpGet:
+            port: couchdb
+            path: /
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        ports: 
+        - containerPort: 5984
+          name: couchdb
```

Comment:
seems you didn't change the name as i mentioned

Reply:
It will be change